images/proxysql/tests/01-version.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+docker run --rm "${IMAGE_NAME}" --version
```

Comment:
We can just put this directly in the `oci_exec` resource now, since this test doesn't do much besides just run the image.